### ISSUE TITLE
bug: can't rename iPhone name under browser extension

### ISSUE THREAD

**Issue by tormento** [twofas/2fas-ios#152]
### Bug type

Other

### App version

5.3.9

### Device environment

iOS 18.0 iPhone 15 Pro Max

### Bug description

Renaming device under browser extension doesn't remember the name.

Plus, it would be nice, at first use, to have the real device name instead of the generic "iPhone".

### Solution

I can't find any.

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I understand that security vulnerabilities should be reported to security@2fas.com instead of on GitHub.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by s3rious0** [comment]
Hey @GrzegorzZajac000 ,


is this a problem with the browser extension instead of iOS?


Greeting

**Comment by GrzegorzZajac000** [comment]
> Hey @GrzegorzZajac000 ,
> 
> 
> is this a problem with the browser extension instead of iOS?
> 
> 
> Greeting

@tormento wrote about name "iPhone" so I suppose that is about phone name under Browser Extension settings in mobile app. In BE user can't change name of the phone.

**Comment by tormento** [comment]
@GrzegorzZajac000 but it's not the real name of my iPhone either. Having two of them, one for work and one for private, makes things a bit uncomfortable.

**Comment by GrzegorzZajac000** [comment]
@tormento yeah I know what the issue is. I only wrote that this is bug in mobile app, not in browser extension ;)
